### PR TITLE
Materials split into lessons and some small changes

### PR DESCRIPTION
**Pull request by apawlik**
I split the materials into lessons. I think it would be good to merge (or review and merge actually) so that the next improvements are on the material organised into split lessons. @waveform80 has just submitted a really useful PR https://github.com/datacarpentry/sql-ecology/pull/33 but that was still to the old file (sql.md) with everything in it. @waveform80 if this PR gets merged, will you be able to re-run your fixes on the individual lessons? Apologies for this!

### CODE REVIEW

**Comment by ethanwhite** [comment]
It looks like #33 has already been merged, but it looks like this will still merge cleanly so that's apparently not an issue.

Can you summarize how this relates to #31. Is this an update version?

**Comment by ethanwhite** [comment]
At a quick glance this looks identical to #31. If you can address the recommendations I made in the earlier PR then we can go ahead and get this in.

**Comment by apawlik** [comment]
@ethanwhite Sure thanks! I'll try to make the changes you suggested in #31 asap. It's not a problem that #33 may merger with conflicts. The problem is that the changes are to sql.md which is supposed to be obsolete (it contains all material rather than the one split to lessons).

**Comment by waveform80** [comment]
Ah, this'll merge cleanly because it's not affecting the original (combined) file at all, so in that sense it'll make the changes I made in #33 obsolete. @apawlik I'm happy to re-do them as another PR after this is merged though (I'd suggest removing `sql.md` shortly after merging this though to avoid confusion - assuming that is that everything from `sql.md` is present in the new files - I haven't checked this thoroughly yet)

**Comment by ethanwhite** [comment]
> The problem is that the changes are to sql.md which is supposed to be obsolete (it contains all material rather than the one split to lessons).

Ah, right, sorry about my confusion.

Sounds like we have a plan. As soon as the changes are in I'll get this merged and @waveform80 can redo the quote fixes. @apawlik you can either include the removal of `sql.md` in the updated PR or I can pull it after this goes in.

**Comment by waveform80** [comment]
Sounds good to me!

**Comment by tracykteal** [comment]
@apawlik can you address the modifications @ethanwhite recommended? It would be really great to get this in, since it's the foundation for any other changes we might want to make. Thanks for putting in this PR!

**Comment by ethanwhite** [comment]
Since this is now holding up other things I'm going to go ahead and merge this PR as it stands and follow it up shortly with a PR fixing the issues I raised in #31.
